subsys/ipc/Kconfig

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020 Nordic Semiconductor (ASA)
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig IPC_SERVICE
```

Comment:
this is coming in as a zephyr subsystem yet it seems to be vendor and hardware specific and selects boards, is this really a generic subsystem or is this something that should be vendor and hardware specific? This probably better fits as an ipc driver rather than a Zephyr subsystem.

Reply:
Our intention is to make this module generic. I believe that with small changes it is possible. We are looking for assistance of IPC experts that could give us hits how to make this module vendor-independent.

Reply:
Rebase the https://github.com/zephyrproject-rtos/zephyr/tree/master/samples/subsys/ipc/openamp samples on rpmsg service and backend should help to verify that the implementation is generic.

Reply:
I don't think it's generic right now, at least because of the Kconfig entries in the file we are commenting. I don't know how to make this configuration generic.

Reply:
> Rebase the https://github.com/zephyrproject-rtos/zephyr/tree/master/samples/subsys/ipc/openamp samples on rpmsg service and backend should help to verify that the implementation is generic

I've created a new sample based on this one. It works correctly with nRF5340-DK and tests shows that passes verification for at least one other board. Seems that IPC Service is generic now